packages/react-native/example/android/app/build.gradle

```diff
@@ -179,3 +179,4 @@ dependencies {
 
 apply from: new File(["node", "--print", "require.resolve('@react-native-community/cli-platform-android/package.json', { paths: [require.resolve('react-native/package.json')] })"].execute(null, rootDir).text.trim(), "../native_modules.gradle");
 applyNativeModulesAppBuildGradle(project)
+apply from: file("../../node_modules/@gutenye/ocr-react-native/ocr.gradle")
```

Comment:
Thanks for the PR. Don't need this, models are in 
 https://github.com/gutenye/ocr/tree/main/packages/react-native/android/src/main/assets folder, they are copied automatically during the build.

Reply:
I see, I'll remove them later, in the latest package I don't get the models embedded to the apps, that's why I created those

Reply:
Are you on Windows? I used a symlink, maybe the symlink is not working for you?

Reply:
> Are you on Windows? I used a symlink, maybe the symlink is not working for you?

nope, I use Arch linux, after looking it up there seems no asset folder on the android folder, that's why the models are not embedded
![image](https://github.com/user-attachments/assets/978ae1f6-2c0d-41d3-a306-13e322ec8076)
It's on v1.4.4 btw

Reply:
@muhamad-rizki I published 1.4.6 version based on https://github.com/gutenye/ocr/pull/17, can you test it and leave a message there?